pilz_trajectory_generation/src/move_group_sequence_service.cpp

```diff
@@ -36,37 +36,37 @@
 
 // Modified by Pilz GmbH & Co. KG
 
-#include "pilz_trajectory_generation/move_group_blend_service.h"
+#include "pilz_trajectory_generation/move_group_sequence_service.h"
 
 #include "pilz_trajectory_generation/capability_names.h"
 #include "pilz_trajectory_generation/command_list_manager.h"
 
 namespace pilz_trajectory_generation
 {
 
-MoveGroupBlendService::MoveGroupBlendService() : MoveGroupCapability("BlendService")
+MoveGroupSequenceService::MoveGroupSequenceService() : MoveGroupCapability("BlendService")
 {
 }
 
-MoveGroupBlendService::~MoveGroupBlendService()
+MoveGroupSequenceService::~MoveGroupSequenceService()
 {
 }
 
-void MoveGroupBlendService::initialize()
+void MoveGroupSequenceService::initialize()
 {
-  blend_manager_.reset(new pilz_trajectory_generation::CommandListManager(ros::NodeHandle("~"),
+  sequence_manager_.reset(new pilz_trajectory_generation::CommandListManager(ros::NodeHandle("~"),
                                                                           context_->planning_scene_monitor_->getRobotModel()));
 
-  blend_service_ = root_node_handle_.advertiseService(BLEND_SERVICE_NAME,
-                                                      &MoveGroupBlendService::plan,
+  blend_service_ = root_node_handle_.advertiseService(SEQUENCE_SERVICE_NAME,
```

Comment:
For the sake of completeness I'd rename this member variable `blend_service_` as well.